### ISSUE TITLE
Visualize how regularization impacts training

### ISSUE THREAD

**Issue by camillamariebach** [camillamariebach/AppliedMachineLearning#24]
Plot train and val loss and acc with and without regularization

Write on spatial dropout
